addons/html_editor/static/src/plugin_sets.js

```diff
@@ -101,6 +102,8 @@ export const COLLABORATION_PLUGINS = [
     CollaborationSelectionAvatarPlugin,
 ];
 
+export const DYNAMIC_PLACEHOLDER_PLUGINS = [DynamicPlaceholderPlugin, QWebPlugin];
```

Comment:
I don't think we should create a constant for that.
There is a constant for the collaboration because it is not obvious that we usually want all of them when we want the collaborative feature.
In the case of the `DynamicPlaceHolder`, if `qweb` is a dependency, the code will crash if the developer does not include it in the array.

But that's a detail, you can resolve the comment if you prefer with the constant.